CacheSim/GenericHashTable.h

```diff
@@ -314,7 +378,7 @@ class GenericHashTable
     const size_t new_capacity = old_capacity ? old_capacity * 2 : 64;
 
     Elem** old_table = m_Table;
-    Elem** new_table = (Elem**) HeapAlloc(m_Heap, 0, sizeof(Elem*) * new_capacity);
+    Elem** new_table = (Elem**) VirtualMemoryAlloc(sizeof(Elem*) * new_capacity);
```

Comment:
I could look at tlsf 3. I do remember it looking promising in the past. Would you want to use that for all the allocation management or just the table itself? (I can't see the perf getting any better than the way I wrote it for the elements since it's pretty trivial except if I just allocate bigger blocks). Using a pooling system for the tables feels a little weird because you're only going to use a given size once, and then you're going to need a contiguous block of a larger size. Unless I'm missing something here?